### PR TITLE
Leeuwenhoek release notes

### PR DESCRIPTION
**Pull request by bullenc**
Leeuwenhoek release notes draft

### CODE REVIEW

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md [hunk 1]
```diff
@@ -26,6 +27,512 @@
...
+		* New permissible value: `Unknown`
+	* Changes made to `undescended_testis_history`
+		* New permissible value: `Unknown`
+	* Changes made to `undescended_testis_history_laterality`
+		* New permissible value: `Unknown`
+	* Removed `maximum` value for properties
+		* `dlco_ref_predictive_percent`
+		* `fev1_fvc_post_bronch_percent`
+		* `fev1_ref_post_bronch_percent`
+		* `fev1_fvc_pre_bronch_percen`
```
Comment (clsackett) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:399]: percent

**Review by clsackett — COMMENTED** [review]
Small typo otherwise LGTM

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md [hunk 2]
```diff
@@ -26,6 +27,512 @@
...
+		* New permissible value: `Not Reported`
+	* Changes made to `oxygen_use_type`
+		* New permissible value: `Unknown`
+		* New permissible value: `Not Reported`
+	* Changes made to `pregnancy_outcome`
+		* New permissible value: `Full Term Birth, NOS`
+	* Changes made to `risk_factor_method_of_diagnosis`
+		* New permissible value: `Unknown`
+	* Changes made to `risk_factors`
+		* Changes made to `items`
```
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:375]: I recommend removing the "items" lines,  I saw a few throughout the notes.

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md [hunk 3]
```diff
@@ -26,6 +27,512 @@
...
+* __Release Date__: May XX, 2024
+
+### New Features and Changes
+
+* New Entity: `germline_mutation_index`
+* New Entity: `submitted_expression_array`
+* Altered `aggregated_somatic_mutation` Entity
+	* New property: `platform`
+* Altered `aligned_reads` Entity
+	* Added `minimum` and/or `maximum` values to properties <!-- summarized this for readability -->
```
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:42]: Make sure this note doesn't appear in the text.  For some reason the syntax highlighting didn't catch this as a comment (as it did the one after).  Highlighting isn't always dependable, but it's worth a check.

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md [hunk 4]
```diff
@@ -26,6 +27,512 @@
...
+	* Changes made to `data_type`
+		* New permissible value: `CNV Model`
+* Altered `copy_number_segment` Entity
+	* New property: `cancer_dna_fraction`
+	* New property: `genome_doubling`
+	* New property: `subclonal_genome_fraction`
+	* New property: `tumor_ploidy`
+	* New property: `tumor_purity`
+* Altered `demographic` Entity
+	* Changes made to `country_of_birth`
```
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:81]: We might want to think about summarizing Not Reported/Unknown values in the notes, but only if we have time.

**Review by wwysoc2 — APPROVED** [review]
Had a few comments, release notes are fine as is, but some would be good to fix if you have time.

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md (hunk 1 above)
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: 👍

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md (hunk 3 above)
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:42]: Yeah I saw this too, in my local version of the markdown file it is greyed out indicating comment. I will double check this when we test QA deployment of docs

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md (hunk 2 above)
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Done 👍

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md (hunk 4 above)
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Summarized props with Not Reported/Unknown values better

File: docs/Data_Submission_Portal/Users_Guide/Best_Practices.md [hunk 5]
```diff
@@ -169,6 +169,7 @@ For any submitter that is uploading scRNA-Seq data, please follow these guidelin
 
 * If the data is single-nuclei RNA-Seq, please populate the associated aliquot field `analyte_type` with `Nuclei RNA`.
 * Please only submit the molecular files as `submitted_unaligned_reads` in FASTQ format.
+* When submitting molecular files, please submit files with file names that are in the acceptable format for CellRanger input. The acceptable format follows this regular expression: SampleName_S[\d*]\_L00[\d]\_R{1,2}\_001.fastq.gz, for example: `SampleName_S1_L001_R1_001.fastq.gz`
```
Comment (wwysoc2) [inline docs/Data_Submission_Portal/Users_Guide/Best_Practices.md:172]: Let's keep a mental note to check the rendering of this regex in markdown.

**Pull request merged**